src/match.js

```diff
@@ -16,7 +16,8 @@ export class Match extends Component {
 		let url = this.nextUrl || getCurrentUrl(),
 			path = url.replace(/\?.+$/,'');
 		this.nextUrl = null;
-		return props.children[0] && props.children[0]({
+		const children = toChildArray(props.children);
+		return children[0] && children[0]({
```

Comment:
This could probably just use `props.children(` to skip the import.
```suggestion
		return props.children({
```